src/feedlistformaction.cpp

```diff
@@ -406,7 +406,9 @@ bool FeedListFormAction::process_operation(Operation op,
 	}
 	break;
 	case OP_MARKALLFEEDSREAD:
-		if (v->confirm(_("Do you really want to mark all feeds as read (y:Yes n:No)? "),
+		if (!cfg->get_configvalue_as_bool(
```

Comment:
Please fix [the formatting issue](https://cirrus-ci.com/task/6433402410762240?command=main#L255) here.

Reply:
just to confirm- so is this the indentation of the pointed line which needs to be fixed?